### PR TITLE
transverse challenge  

### PR DESCRIPTION
**Pull request by mykelgozie**
Please bond i hard some challenges, I still tried to complete my challenge although its late .

### CODE REVIEW

File: src/word-search/index.js [hunk 1]
```diff
@@ -1,3 +1,189 @@
-function wordSearch(words, word) {}
+function wordSearch(words, word) {
```
Comment (flinchy) [inline src/word-search/index.js:1]: format your code properly, remove the unnecessary white spaces

File: src/count-islands/index.js [hunk 2]
```diff
@@ -1,3 +1,114 @@
-function countIslands(grid) {}
+function countIslands(grid) {
+
```
Comment (flinchy) [inline src/count-islands/index.js:2]: format your code properly, remove the unnecessary white spaces

File: src/word-search/index.js (hunk 1 above)
Comment (mykelgozie) [reply src/word-search/index.js:1]: aiit i would do just that. thanks

Comment (mykelgozie) [reply src/word-search/index.js]: i have done that .

File: src/count-islands/index.js (hunk 2 above)
Comment (mykelgozie) [reply src/count-islands/index.js:2]: i have removed the white spaces .

File: src/count-islands/index.js [hunk 3]
```diff
@@ -1,3 +1,114 @@
...
+                checkIsland(grid , c, d, visit);
+
+                count = count + 1;
+
+
+                
+            }
+           
+
+
```
Comment (flinchy) [inline src/count-islands/index.js:47]: I can still see some whitespaces

Comment (mykelgozie) [reply src/count-islands/index.js]: done removed whites spaces

File: src/count-islands/index.js (hunk 2 above)
Comment (flinchy) [reply src/count-islands/index.js:2]: if you use vscode editor, you can use prettier plugin to format your code

**Pull request merged**